src/dispatch/case/messaging.py

```
@@ -77,6 +77,10 @@ def send_case_triage_reminder(case: Case, db_session: SessionLocal):
         }
     ]
 
+    if not case.assignee:
```

nit, we can fail faster above `items`.